docs/source/Tutorials/interface_plots.ipynb

```diff
@@ -0,0 +1,668 @@
+    "params = {\n",
+    "    \"beta\": 0.98,  # discount factor\n",
+    "    \"delta\": 1.00,  # disutility of work\n",
+    "    \"constant\": np.log(20.0),  # age-independent labor income\n",
+    "    \"rho\": 1.00,  # coefficient of relative risk aversion\n",
+    "    \"lambda\": 2e-16,  # taste shock (scale) parameter\n",
+    "    \"interest_rate\": 0.00,  # interest rate on capital\n",
+    "    \"consumption_floor\": 0.5,  # consumption floor/retirement safety net\n",
+    "    ## Parameters specific to the more complex dc-egm retirement model -- can be ignored for now ##\n",
+    "    \"exp\": 0.00,  # return to experience\n",
+    "    \"exp_squared\": 0.0,  # return to experience squared\n",
+    "    \"sigma\": 0.0,  # shock on labor income sigma parameter/standard deviation\n",
+    "    \"income_shock_mean\": 0.0,  # shock on labor income mean parameter\n",
+    "}\n",
+    "model_params = {\n",
+    "    \"n_periods\": 43,  # number of periods - 1 since we start counting at 0\n",
+    "    ## Parameters specific to the more complex dc-egm retirement model -- can be ignored for now ##\n",
+    "    \"min_age\": 20,  # minimum age\n",
+    "    \"n_quad_points_stochastic\": 75,  # number of quadrature points for stochastic integration\n",
+    "}\n",
+    "options = {\n",
+    "    \"model_params\": model_params,\n",
+    "    \"state_space\": {\n",
+    "        \"n_periods\": 43,  # number of periods T\n",
+    "        \"choices\": np.array([0, 1]),  # choices d\n",
+    "        \"continuous_states\": {  # continuous decision state\n",
+    "            \"wealth\": jnp.linspace(  # wealth grid e.g. equally spaced grid\n",
+    "                0,  # minimum wealth\n",
+    "                150,  # maximum wealth\n",
+    "                500,  # min. number of grid points\n",
+    "            )\n",
+    "        },\n",
+    "    },\n",
+    "    \"tuning_params\": {\n",
+    "        \"fues_n_points_to_scan\": 20,  # number of points to scan for the fues\n",
+    "        \"fues_jump_thresh\": 8,  # threshold for the fues jump\n",
+    "    },\n",
+    "}\n",
+    "# load the model functions for the exact functions see the example model file\n",
+    "model_funcs = load_example_models(\"dcegm_paper\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solving the Model\n",
+    "\n",
+    "The simplest approach to setting up the model is using the `setup_model` function and obtaining the solve function with `get_solve_func_for_model`. The model is then solved using the `solve` function, which returns the solved model containing three main components:\n",
+    "- `value`: An array of values of the choice-specific value functions for each state. For a detailed description of the choice-specific value function, see the XXX.\n",
+    "- `policy`: An array of choice-specific consumption functions for each state. For a detailed description of the policy function, see XXX\n",
```

Comment:
Missing reference for `value`, `policy`, `wealth_grid`, e.g., here:

- `value`: An array of values of the choice-specific value functions for each state. For a detailed description of the choice-specific value function, see the **XXX**

Reply:
Deleted reference